src/Connections/HttpConnection.cs

```diff
@@ -45,7 +47,7 @@ private bool IsAuthenticated(HttpRequestEventArgs e)
         {
             var req = e.Request;
 
-            if (!string.IsNullOrEmpty(authentication) && !req.Headers.Contains("Authentication"))
+            if (!string.IsNullOrEmpty(authentication) && !req.Headers.ContainsKey("Authentication"))
```

Comment:
I added the ContainsKey because I can't get it to build a Solution (even using your unaltered HttpConnection.cs file).  Again, I'm new and learning lol so could just be a *me* problem...but I get this Error when attempting to just use `Contains` vice `ContainsKey`

**CS1929:**	```'NameValueCollection' does not contain a definition for 'Contains' and the best extension method overload 'Queryable.Contains<string>(IQueryable<string>, string)' requires a receiver of type 'System.Linq.IQueryable<string>'	7DTDWebsockets	C:\7DTD-Websocket\src\Connections\HttpConnection.cs	45	Active```

Reply:
@MustachedManiac Interesting, I havent run into that issue. If that is the case, then using `req.Headers.AllKeys.Contains` inline. As it is the only instance of this of checking headers, there is no need to make a utility class for it.

Reply:
@KK964 `AllKeys.Contains` worked like a charm, thank you! (it learnnnnns 🤣 )